README.md

```diff
@@ -67,6 +67,10 @@ We are more than excited to be a part of [Kharagpur Winter of Code](https://kwoc
   ```bash
   > pipenv shell
   ```
+  
+* Create a file named `.env` in project's root directory (i.e. inside **pro1**, where `.env.example` reside)
```

Comment:
Check out these changes and mention them in #52 . Will be taken care therein

Reply:
Ok